autoload/copilot.vim

```diff
@@ -171,7 +171,7 @@ function! copilot#NvimNs() abort
   return nvim_create_namespace('github-copilot')
 endfunction
 
-function! copilot#Clear() abort
```

Comment:
The `Clear` function doesn't unlet `b:_copilot` anymore, and is separated into `CancelOngoingRequest` and `UpdatePreview`.